### ISSUE TITLE
Code block is not working in ubuntu 16.04 

### ISSUE THREAD

**Issue by DulanGit** [klauscfhq/tusk#74]
# Reporting an Issue

Thank you for taking the time to contribute to Tusk! ✨🎉

## Suggested quick smart steps

- Run through the [contributing guidelines](https://github.com/klauscfhq/tusk/blob/master/contributing.md)
- Support questions are better asked in the project's [chat](https://gitter.im/klauscfhq/tusk). We will get back at you in a flash.
- Ensure the issue is not already reported

*Delete the above section and the content in the sections below before submitting your issue*

## Description

Please provide as much *relevant* info as possible

### Error Message (if any)

```
Copy the error message here
```

## Relevant Links (if any)

- Any link to an other project that you believe it might be relevant

## Environment

Tell us which operating system you are using, as well as which versions of [Tusk](https://github.com/klauscfhq/tusk/releases/latest).
Run the following script to get your OS info quickly.

```bash
node -e "const osInfo = require('os'); console.log('\n' + 'OS Type: ' + '\t' + osInfo.type() + '\n' + 'OS Platform:' + '\t' + osInfo.platform() + '\n' + 'OS Release: ' + '\t' + osInfo.release() + '\n' + 'Architecture: ' + '\t' + osInfo.arch() + '\n');"
```

## Additional Information

Include any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by kaaljabr** [comment]
And it does not work on MAC OS either.

**Comment by klauscfhq** [comment]
It looks like an issue on Evernote side, the `code-block` option disappears from their editor when running it on Chrome. Visit their [issue tracker](https://discussion.evernote.com/topic/112510-why-have-all-code-blocks-disappeared/) for more info.

**Comment by batjko** [comment]
Yes, it's been an issue at least since March this year (https://discussion.evernote.com/topic/112445-how-do-i-add-code-blocks/), and Evernote team don't seem to give it any kind of priority.
